### PR TITLE
[Dashboard] Remove ethers from list-form

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR focuses on updating the currency conversion logic in the `CreateListingsForm` component.

### Detailed summary
- Updated currency conversion using `toUnits` and `toWei` functions
- Added handling for different currency contract addresses
- Removed `mulDecimalByQuantity` function

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
* **#4241** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4241?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx [hunk 1]
```diff
@@ -213,14 +212,12 @@ export const CreateListingsForm: React.FC<CreateListingsFormProps> = ({
...
-            buyoutBidAmount: mulDecimalByQuantity(
-              formData.buyoutPricePerToken,
-              formData.quantity,
-            ),
+            minimumBidAmount: (
+              BigInt(formData.reservePricePerToken) * BigInt(formData.quantity)
+            ).toString(),
+            buyoutBidAmount: (
+              BigInt(formData.buyoutPricePerToken) * BigInt(formData.quantity)
+            ).toString(),
```
Comment (jnsdls) [inline apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx:220]: this won't work if values are decimals, probably need to parse the prices and the quantity can be simply bigint -> then multiply, boom

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx (hunk 1 above)
Comment (kien-ngo) [reply apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx:220]: yesser correct! I switched to the amount-in-wei. Did it on the plane, just got the chance to submit now

File: packages/thirdweb/src/react/web/ui/prebuilt/NFTProvider/index.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+"use client";
```
Comment (jnsdls) [inline packages/thirdweb/src/react/web/ui/prebuilt/NFTProvider/index.tsx:1]: @kien-ngo why is this in here? :)

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: packages/thirdweb/src/react/web/ui/prebuilt/NFTProvider/index.tsx (hunk 2 above)
Comment (kien-ngo) [reply packages/thirdweb/src/react/web/ui/prebuilt/NFTProvider/index.tsx:1]: omg

**Review by jnsdls — APPROVED** [review]

**Pull request merged**